src/components/AmountWithoutCurrencyInput.tsx

```diff
@@ -104,4 +104,4 @@ function AmountWithoutCurrencyInput({
 
 AmountWithoutCurrencyInput.displayName = 'AmountWithoutCurrencyInput';
 
-export default AmountWithoutCurrencyInput;
+export default React.forwardRef(AmountWithoutCurrencyInput);
```

Comment:
I think we need to go back to this:
```
function AmountWithoutCurrencyInput(
    {value: amount, shouldAllowNegative = false, inputID, name, defaultValue, accessibilityLabel, role, label, onInputChange, ...rest}: AmountFormProps,
    ref: ForwardedRef<BaseTextInputRef>,
) {
```

But add the new props that has been added